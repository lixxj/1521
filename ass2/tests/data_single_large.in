a = malloc 6000
b = malloc 1000
free b
free a