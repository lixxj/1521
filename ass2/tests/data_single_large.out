test3: couldn't allocate 6000 bytes for `a'
+00000 (F, 5000) 
